node/service/src/client.rs

```diff
@@ -57,13 +57,15 @@ impl sc_executor::NativeExecutionDispatch for VaraExecutorDispatch {
         gear_ri::gear_ri::HostFunctions,
         gear_ri::sandbox::HostFunctions,
         sp_crypto_ec_utils::bls12_381::host_calls::HostFunctions,
+        gear_ri::gear_bls_12_381::HostFunctions,
```

Comment:
To be consistent in naming with sp_crypto
```suggestion
        gear_ri::gear_bls12_381::HostFunctions,
```

Reply:
This is an auto-generated name by `#[runtime_interface]` macro